server/text_generation_server/models/flash_causal_lm.py

```diff
@@ -820,6 +825,34 @@ def warmup(self, batch: FlashCausalLMBatch):
             self.device,
         )
 
+        if SYSTEM == "rocm":
+            if os.environ.get("PYTORCH_TUNABLEOP_ENABLED", False):
+                if os.environ.get("PYTORCH_TUNABLEOP_TUNING", "1"):
+                    torch.cuda.tunable.tuning_enable(True)
+
+                if os.environ.get("PYTORCH_TUNABLEOP_SEQLENS", False):
+                    tuning_sequences = [int(val) for val in os.environ["PYTORCH_TUNABLEOP_SEQLENS"].split(",")]
+                else:
+                    tuning_sequences = [1, 2, 4, 8, 16, 32]
```

Comment:
Couldn't we reuse CUDA_GRAPHS env variable here ? (I'm ok to rename the internal variable if the thing is not CUDA specific anymore, the cli flag will probably remain the same because it's user-level).

Reply:
yes